src/accounts/views.py

```diff
@@ -79,9 +94,20 @@ def profile_view(request):
             {
                 "Id": item.get("Id"),
                 "Name": item.get("Name", "No Name"),
-                "Category": item.get("Category", "N/A"),
-                "Distance": "N/A",
                 "Address": item.get("Address", "N/A"),
+                "Lat": float(item.get("Lat")) if item.get("Lat") else None,
+                "Log": float(item.get("Log")) if item.get("Log") else None,
+                "Ratings": (
+                    "N/A"
+                    if item.get("Ratings") in [0, "0", None]
+                    else str(item.get("Ratings"))
+                ),
+                "RatingCount": str(item.get("rating_count", 0)),
+                "Category": item.get("Category", "N/A"),
+                "MapLink": f"https://www.google.com/maps/dir/?api=1&destination={quote(item.get('Address'))}",
+                "Distance": "N/A",  # Since distance may not be available
```

Comment:
nit: let's remove comments that are self-explanatory

Reply:
done in most recent push